pkg/trace/config/config.go

```diff
@@ -107,6 +108,9 @@ type AgentConfig struct {
 	// It maps tag keys to a set of replacements. Only supported in A6.
 	ReplaceTags []*ReplaceRule
 
+	// AdditonalMetadata list metadata that will be added to all spans
```

Comment:
```suggestion
	// GlobalTags lists tags that will be attached to all spans.
```
Can we please call it GlobalTags? This is simply because we are already using this terminology for the same functionality in tracers. It will be easier to follow.